### ISSUE TITLE
[Suggestion] Add "What every programmer should know about memory" by Ulrich Drepper for reading

### ISSUE THREAD

**Issue by ujos** [dendibakh/perf-book#5]
I was really surprised not seeing the Ulrich Drepper's book "What every programmer should know about memory" in the References chapter. It was written in 2007 but the information explained there is still actual.

I believe it worth to add it to the References list for next reading.

**Comment by paulfloyd** [comment]
Nah. It's not that good despite the pretentious title and it goes into a lot of unnecessary detail. CPUs _have_ changed a lot since then (e.g., with chiplets).
